aura.grammar

```diff
@@ -0,0 +1,56 @@
+
+use_statement: identifier ['as' identifier]
+
+function_def: 'fn' NAME '(' decl_list ')' type NEWLINE function_body
```

Comment:
Is it `NAME` or `identifier`? I'm leaning towards the latter, since functions have a type.